core/controllers/base.py

```diff
@@ -164,6 +174,7 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
 
         self.user_id = None
         self.username = None
+        self.email = None
```

Comment:
Why is this change needed, and is there a test to verify it?

Reply:
This change is used to grant `feconf.ADMIN_EMAIL_ADDRESS` the unique privilege of running the Grant/Revoke Firebase super-admin privileges controllers.

There are no tests for checking attributes on the Base controller, but this is tested regardless by the Grant/Revoke super-admin privileges controller.

Reply:
Ack, thanks.